src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -102,6 +102,29 @@ public static TypedData ToRpc(this object value)
                     }
                 }
 
+                // parse ClaimsPrincipal if exists
+                if (request.HttpContext?.User?.Identities != null)
+                {
+                    foreach (var id in request.HttpContext.User.Identities)
```

Comment:
Add logging. This will help when looking through Kusto

Reply:
Added.